lib/RAST_SDK/RAST_SDKImpl.pm

```diff
@@ -509,8 +513,9 @@ sub annotate {
 			my $feature = $inputgenome->{features}->[$i];
 			if ($feature->{type} eq "gene" and defined($feature->{protein_translation})) {
 				$feature->{type} = "CDS"
-			}
+			};
 		}
+		delete $inputgenome->{feature_counts};
```

Comment:
Why is this done?

Reply:
I'll add a comment to clarify. Basically I'm pulling in the new genomes without downgrading them (to avoid losing information) but when RAST annotates the genome it becomes incompatible with the new spec. Removing this attribute triggers an "upgrade" to the genome that fixes these problems when saving with GFU